apps/token/src/components/locked-progress/locked-progress.tsx

```diff
@@ -1,11 +1,57 @@
-import './locked-progress.scss';
-
 import React from 'react';
-
 import { Colors } from '../../config';
 import { formatNumber } from '../../lib/format-number';
 import type { BigNumber } from '../../lib/bignumber';
 
+const ProgressContents = ({
+  light,
+  children,
+}: {
+  light: boolean;
+  children: React.ReactNode;
+}) => (
+  <div
+    className={`flex justify-between py-2 font-mono ${
```

Comment:
suggestion (non-blocking): This would be cleaner using the classnames function which we already use in other projects. Same goes for any other files where we are nesting ternaries in strings